invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -82,3 +82,22 @@ code {
     font-weight: normal;
     font-style: normal;
 }
+
+/*
+* Examples:
+* className="mt-10"
+* className="mr-5"
+* className="ml-20"
+* className="mb-30"
+*/
+.generate-margins(@max, @direction, @alias, @i: 5) when (@i =< @max) {
+  .m@{alias}-@{i} {
+    margin-@{direction}: unit(@i, px) !important;
+  }
+  .generate-margins(@max, @direction, @alias, (@i + 5));
+}
+
+.generate-margins(30, right, r);
```

Comment:
Can you add also here the generated classes that will be created from this function call. F.e:
```
The following classses will be generated:
.mr-{5,10,15,20,25,30} {
    margin-right {
      {5,10,15,20,25,30}px !important;
    }
}
```
Also, I think we should remove !important because it will make overridability impossible...

Reply:
If !important is removed it won't work